arangod/GeneralServer/RestHandler.h

```diff
@@ -233,6 +236,11 @@ class RestHandler : public std::enable_shared_from_this<RestHandler> {
 
   RequestLane _lane;
 
+  std::shared_ptr<LogContext::Values> _logContextScopeValues;
+  LogContext::EntryPtr _logContextEntry;
+  static constexpr char UrlName[] = "url";
+  static constexpr char UserName[] = "username";
```

Comment:
Didn't we say all the parameters should be placed in a central header file?

Reply:
Addressed in latest commit